docs/crashlytics/usage.mdx

```diff
@@ -3,15 +3,47 @@ title: Using Firebase Crashlytics
 sidebar_label: Usage
 ---
 
+## Getting started
+
+Enable Forebase Crashytlics in the Firebase console.
+
+![Crashytlics - Enable in the console](enable-firebase-crashlytics.png)
+
+Until an error has been reported, you will see this screen.
+
+![Crashytlics init screen](crashlytics-init.png)
+
 To start using Firebase Crashlytics within your project, import it at the top of your project files:
 
 ```dart
 import 'package:firebase_crashlytics/firebase_crashlytics.dart';
 ```
 
+Run an example, such as...
+
+```dart
+//Init application
+await Firebase.initializeApp();
+
+//Elsehwere in your code
+FirebaseCrashlytics.instance.crash();
+```
+
+This will restart the application. If using an emulator, please re-run...
+
+```dart
+  flutter run
+```
+
+`or restart the app manually through the emulator.`
```

Comment:
Remove this bit, not needed?

Reply:
This is a funny one. At times I needed to run one or the other. I kept it in for now as a backup, in case `flutter run` failed.

Open to ideas though!

Reply:
You do have to re-open the app after a crash() call as it only submits on app startup, calling run again re-installs the apk afaik and so loses previous crash data?

Reply:
When I was firing errors yesterday, I had to crash() and restart app using `flutter run` for reports to send. At least, that was the method I was using for it to work consistently.